dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -872,11 +870,16 @@ private boolean doesOrganizationExistToUserResourceDAO(Long organizationId, Long
      * @param userId
      * @return True if organization exists to user, false otherwise
      */
-    static boolean doesOrganizationExistToUser(Long organizationId, Long userId, OrganizationDAO organizationDAO) {
+    static boolean doesOrganizationExistToUser(Long organizationId, Long userId, OrganizationDAO organizationDAO, UserDAO userDAO) {
         Organization organization = organizationDAO.findById(organizationId);
         if (organization == null) {
             return false;
         }
+        // Admins and curators should be able to see unapproved organizations
```

Comment:
They can see all orgs, not just unapproved ones that they don't belong to.

No need to change though.

Reply:
Codacy appears to be upset because my test method doesn't contain JUnit asserts/fails (it repeatedly calls a subtest method that invokes them).